src/Raven.Server/Documents/Replication/ChangeVectorParser.cs

```diff
@@ -274,6 +274,85 @@ public static void MergeChangeVector(string changeVector, List<ChangeVectorEntry
             ThrowInvalidEndOfString(state.ToString(), changeVector);
         }
 
+        public static void MergeChangeVectorDown(string changeVector, List<ChangeVectorEntry> entries)
+        {
+            var start = 0;
+            var current = 0;
+            var state = State.Tag;
+            int tag = -1;
+
+            while (current < changeVector.Length)
+            {
+                switch (state)
+                {
+                    case State.Tag:
+                        if (changeVector[current] == ':')
+                        {
+                            tag = ParseNodeTag(changeVector, start, current - 1);
+                            state = State.Etag;
+                            start = current + 1;
+                        }
+                        current++;
+                        break;
+                    case State.Etag:
+                        if (changeVector[current] == '-')
+                        {
+                            var etag = ParseEtag(changeVector, start, current - 1);
+
+                            if (current + 23 > changeVector.Length)
```

Comment:
What is `23` here? Let's use a meaningful constant

Reply:
Done